### PR TITLE
Added support for Messaging.isSupported

### PR DESCRIPTION
**Pull request by martinory**
Hi!
According to [Firebase docs -- firebase. messaging](https://firebase.google.com/docs/reference/js/firebase.messaging#functions_1):
**Functions**
   isSupported
   isSupported(): boolean
   Returns boolean

The function is accessed in js by firebase.messaging.isSupported(), here as a static member of the Messaging class (Messaging.isSupported()).
There are no tests for Messaging.

Dart VM version: 2.4.0 (Wed Jun 19 11:53:45 2019 +0200) on "windows_x64"

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -22,4 +22,4 @@ dev_dependencies:
 
   build_runner: ^1.0.0
   build_test: ^0.10.0
-  build_web_compilers: '>=1.2.0 <3.0.0'
+  build_web_compilers: '>=1.2.0 <3.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: revert this change!

File: lib/src/messaging.dart [hunk 2]
```diff
@@ -16,6 +16,8 @@ class Messaging extends JsObjectWrapper<messaging_interop.MessagingJsImpl> {
     return _expando[jsObject] ??= Messaging._fromJsObject(jsObject);
   }
 
+  static bool isSupported() => messaging_interop.isSupported();
```
Comment (kevmoo) [inline lib/src/messaging.dart]: Did you test this locally?

Comment (martinory) [reply lib/src/messaging.dart]: I did test this locally on the computer as well as on my hosted web app on an iPad where I got true and false respectively.

File: pubspec.yaml (hunk 1 above)
Comment (martinory) [reply pubspec.yaml]: I'm not accustomed to git but I tried to use Git Gui to amend the change. It's not visible [here](https://github.com/martinory/firebase-dart/commit/c0011bcda02931403dd02faf03c82b9e19d57c82) anymore.

Comment (kevmoo) [reply pubspec.yaml]: > I'm not accustomed to git but I tried to use Git Gui to amend the change. It's not visible [here](https://github.com/martinory/firebase-dart/commit/c0011bcda02931403dd02faf03c82b9e19d57c82) anymore.

But now there are a LOT of other files! Uh...
